cmake/EthUtils.cmake

```diff
@@ -22,3 +22,43 @@ macro(replace_if_different SOURCE DST)
 	endif()
 endmacro()
 
+macro(eth_add_test NAME) 
+
+	# parse arguments here
+	set(commands)
+	set(current_command "")
+	foreach(arg ${ARGN})	
```

Comment:
I don't not sure if CodingStandards to cmake files. But I agree, that it looks better with a space ;)